modules/get-started/pages/whats-new.adoc

```diff
@@ -11,22 +11,33 @@ See also: xref:deploy:deployment-option/cloud/whats-new-cloud.adoc[]
+xref:manage:security/authorization/rbac.adoc[Role-based access control] (RBAC) provides an extension to Redpanda's existing access control lists (ACLs). Using `rpk` or Redpanda Console, you can create roles and assign those roles as the authorized or denied party for an ACL. Every user you assign to that role inherits those authorizations. RBAC provides a more flexible and efficient way to manage user permissions, especially with complex organizational hierarchies or large numbers of users.  
 
 == Data transforms
 
-* Output to multiple topics: Data transforms can now output to multiple topics, for use cases that might require distributing different message types to multiple event streams, or for writing valid records to an output topic and invalid records to a dead-letter queue. See the xref:reference:data-transform-golang-sdk.adoc[Go] or xref:reference:data-transform-rust-sdk.adoc[Rust] SDK reference for data transforms for implementation details.
+* Output to multiple topics: Data transforms can now output to multiple topics.  For example, you can distribute different message types to multiple event streams, or you can write valid records to an output topic and invalid records to a dead-letter queue. See the xref:reference:data-transform-golang-sdk.adoc[Golang SDK] and xref:reference:data-transform-rust-sdk.adoc[Rust SDK] for data transforms.
 
-* xref:develop:data-transforms/run-transforms.adoc#view-data-transform-logs[Data transform logs] are now stored in an internally-managed topic and are accessible via `rpk` and the Redpanda Console.
+* xref:develop:data-transforms/run-transforms.adoc#view-data-transform-logs[Data transform logs] are now stored in an internally-managed topic and are accessible via `rpk` and Redpanda Console.
 
 == Topic-aware partition balancing 
 
 Redpanda now prioritizes xref:manage:cluster-maintenance/cluster-balancing.adoc[balancing a topic's partition replica count on each broker] while it's balancing the cluster's overall partition count. Because different topics in a cluster can have vastly different load profiles, this better distributes the workload evenly across brokers. This feature is enabled by default with the `partition_autobalancing_topic_aware` property. After upgrading to 24.1, new topics are distributed in a topic-aware fashion, and the next partition rebalance (triggered by adding new brokers or by calling the `/v1/partitions/rebalance` Admin API endpoint) evens out the partition counts on each broker.
```

Comment:
1. 'balancing a topic's partition replica count evenly across all brokers while it's ....'
2. 'this better distributes the workload of the most heavily used topics evenly across brokers'

3.
> " and the next partition rebalance (triggered by adding new brokers or by calling the `/v1/partitions/rebalance` Admin API endpoint)"

or by continuous data balancing?